pkg/network/protocols/http2/protocol.go

```diff
@@ -431,6 +431,10 @@ func (p *Protocol) GetStats() *protocols.ProtocolStats {
 	}
 }
 
+// ReleaseStats releases stats objects.
+func (p *Protocol) ReleaseStats() {
```

Comment:
HTTP2 uses the same statsKeeper as HTTP, so according the mechanism, it should be released